datadog/resource_datadog_dashboard.go

```diff
@@ -4019,6 +4019,14 @@ func getScatterplotDefinitionSchema() map[string]*schema.Schema {
 							Schema: getScatterplotRequestSchema(),
 						},
 					},
+					"scatterplot_table": {
+						Description: "Scatterplot request containing formulas and functions.",
+						Type:        schema.TypeList,
+						Optional:    true,
+						Elem: &schema.Resource{
+							Schema: getScatterplotRequestSchema(),
```

Comment:
This should be its own separate schema, with only formulas, queries and response format, as per the [docs](https://docs.datadoghq.com/dashboards/widgets/scatter_plot/#api).

Reply:
Resolved in [e8e0](https://github.com/DataDog/terraform-provider-datadog/pull/1275/commits/9ecadcc1d889605b30f5de449d7098231ac2e8e0).